src/pip/_internal/download.py

```diff
@@ -48,6 +48,7 @@
 
 try:
     import ssl  # noqa
+
```

Comment:
These newline changes seem unrelated... Could you remove them or is there some reason I'm not seeing?